tests/test_characteristics/test_volume.py

```diff
@@ -0,0 +1,54 @@
+from unittest import TestCase
+
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from foapy.characteristics.volume import volume
+
+
+class TestVolume(TestCase):
+    """
+    Test list for volume calculate
+    """
+
+    def test_calculate_start_lossy_volume(self):
+        X = ["B", "B", "A", "A", "C", "B", "A", "C", "C", "B"]
+        expected = np.array([144])
+        exists = volume(X, 1, 1)
+        assert_array_equal(expected, exists)
+
+    def test_calculate_start_normal_volume(self):
+        X = ["B", "B", "A", "A", "C", "B", "A", "C", "C", "B"]
+        expected = np.array([2160])
+        exists = volume(X, 1, 2)
```

Comment:
Please use enumerations to improve readability. 
```suggestion
        exists = volume(X, binding.start, mode.normal)
```